books/free-programming-books-my.md

```diff
@@ -1,33 +1,61 @@
 ### Index
 
+* [API](#api)
+* [Blockchain](#blockchain)
 * [Go](#golang)
-* [HTML](#html)
+* [HTML/CSS](#html)
+* [JavaScript](#javascript)
 * [Linux](#linux)
+* [PHP](#php)
 * [Python](#python)
 * [Web Development](#web-development)
 
 
+### <a id="api"></a>API
+
+* [API - On Point](https://eimaung.com/api/) - Ei Maung
```

Comment:
It's not suitable add a new category to fit a lonely resource. Maybe this could be transfer to Javascript or NodeJS